docs/source/get_started/kedro_concepts.md

```diff
@@ -90,7 +90,7 @@ By default, Kedro creates one file, `credentials.yml`, in `conf/local`.
 
 ### `data`
 
-The `data` folder contains multiple subfolders to store project data. We recommend you put raw data into `raw` and move processed data to other subfolders according to the [data engineering convention](../faq/faq.md#what-is-data-engineering-convention).
+The `data` folder contains multiple subfolders to store project data. We recommend you put raw data into `raw` and move processed data to other subfolders according to the [data engineering convention](https://towardsdatascience.com/the-importance-of-layered-thinking-in-data-engineering-a09f685edc71).
```

Comment:
The sentence "The data engineering convention" has always bugged me, like there's only one. In any case, let's tackle this along with https://github.com/kedro-org/kedro-devrel/issues/42 ?